src/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Args, getTasks, Quest } from "grimoire-kolmafia";
+import args from "./args";
+import { questStep } from "libram";
+import { CandyTask } from "./lib";
+import CandyEngine from "./engine";
+import GLOBAL_TASKS from "./regularTasks";
+import TRICK_TREAT_TASKS from "./trickTreatTasks";
+import { myAdventures } from "kolmafia";
+
+export default function main(argstring = ""): void {
+  Args.fill(args, argstring);
```

Comment:
Print help text if `freecandy help` is called
```suggestion
  Args.fill(args, argstring);
  if (args.help) {
    Args.showHelp(args);
    return;
  }
```